server/bundler/babel/babel-lodash-es/test/index.js

```diff
@@ -13,27 +13,27 @@ describe( 'babel-lodash-es', () => {
 	}
 
 	test( 'should transform named import from top-level package', () => {
-		const code = "import { get } from 'lodash-es'";
-		expect( transform( code ) ).toMatchSnapshot();
+		const code = transform( "import { get } from 'lodash-es';" );
+		expect( code ).toBe( "import { get } from 'lodash';" );
```

Comment:
thanks for doing this!

I agree with @dmsnell that its clearer without snapshots in this case.
Snapshots are pretty awesome though :)